security/file_integrity_operator/file-integrity-operator-installation.adoc

```diff
@@ -0,0 +1,11 @@
+[id="installing-file-integrity-operator"]
+= Installing the File Integrity Operator
+include::modules/common-attributes.adoc[]
+:context: file-integrity-operator-installation
+
+toc::[]
+
+
+include::modules/file-integrity-operator-installing-web-console.adoc[leveloffset=+1]
+
+include::modules/file-integrity-operator-installing-cli.adoc[leveloffset=+1]
```

Comment:
We also get some PRs about the installation on disconnected clusters cluster. Could we put a link here to indicate customer how to install with disconnected cluster?
` for disconnected clusters cluster, please refer to https://docs.openshift.com/container-platform/4.6/operators/admin/olm-restricted-networks.html`